jenkins/databricks/test.sh

```diff
@@ -36,11 +61,37 @@ sudo "$(which pip)" install pytest sre_yield requests pandas pyarrow findspark p
 export SPARK_HOME=/databricks/spark
 # change to not point at databricks confs so we don't conflict with their settings
 export SPARK_CONF_DIR=$PWD
-export PYTHONPATH=$SPARK_HOME/python:$SPARK_HOME/python/pyspark/:$SPARK_HOME/python/lib/py4j-0.10.9-src.zip
-if [[ $BASE_SPARK_VERSION == "3.2.1" ]]
+# Get Python version from environment after lowercase and remove minor.
+# Something like python3.8 for DB10.4 and python3.9 for DB11.3
+sw_versions[PYTHON]=$(python --version | sed 's/[A-Z]/\L&/g;s/ //g' | cut -d . -f 1,2)
+# set Iceberg related versions. See https://iceberg.apache.org/multi-engine-support/#apache-spark
+# Spark 3.3 | Initial Iceberg Version 0.14.0 | Latest Support 1.0.0
```

Comment:
not sure what these comments mean, they don't match the versions below for 0.14.1, 0.13.2, etc..  so not sure why we need these?

Reply:
I can remove that table that shows the minimal version supported for each spark version.